Docs: updating mentions of Littlepay with transit processor in product docs (#3021)
cal-itp/benefits: docs/README.md

The user interface and content is available in both English and Spanish. Additional language support is possible via Django's
[i18n and l10n features][i18n].

> The application communicates with external vendor services like [Littlepay][littlepay] and [Switchio](https://switchio.com/transport/) via API calls and others like the [Identity Gateway](https://dev.auth.cdt.ca.gov) via redirects, both over the public internet. See [all the system interconnections][interconnections].

Author: Updated CDT link to this URL.

> The application communicates with external vendor services like [Littlepay][littlepay] and [Switchio](https://switchio.com/transport/) via API calls and others like the [Identity Gateway](https://dev.auth.cdt.ca.gov) via redirects, both over the public internet. See [all the system interconnections][interconnections].

Reviewer: Looks like you updated the wrong link, Andy. You updated the one in the paragraph below, not this paragraph Machiko commented on.

The user interface and content is available in both English and Spanish. Additional language support is possible via Django's
[i18n and l10n features][i18n].

> The application communicates with external [transit processors vendors](#transit-processors) via API calls and others like the [Identity Gateway](https://dev.auth.cdt.ca.gov) via redirects, both over the public internet. See [all the system interconnections][interconnections].

Reviewer: Previous thread got marked outdated, so here's a fresh suggestion:

```suggestion
The application communicates with external [transit processor vendors](#transit-processors) via API calls and others like the [Identity Gateway](https://digitalidstrategy.cdt.ca.gov/primary-elements.html) via redirects, both over the public internet. See [all the system interconnections][interconnections].
```

### Infrastructure

The Benefits application is deployed to Microsoft Azure. Traffic is encrypted between the user and the application, as well as between the application and external systems.

> The network is managed by the [California Department of Technology (CDT)](https://digitalidstrategy.cdt.ca.gov/primary-elements.html), who provide a firewall and [distributed denial-of-service (DDoS)](https://www.cloudflare.com/learning/ddos/what-is-a-ddos-attack/) protection.

Reviewer: Part two of the above, restoring the original link here.

```suggestion
The network is managed by the [California Department of Technology (CDT)](https://cdt.ca.gov/), who provide a firewall and [distributed denial-of-service (DDoS)](https://www.cloudflare.com/learning/ddos/what-is-a-ddos-attack/) protection.
```

Sensitive user information exists in the following places:

- To enroll for reduced fares as an older adult, U.S. Veteran, or Calfresh cardholder, riders need to [provide personal information to Login.gov](https://benefits.calitp.org/help#login-gov-verify).
- To enroll for reduced fares as a Medicare cardholder, riders need to [provide personal information to Medicare.gov](https://www.medicare.gov/basics/get-started-with-medicare).

> - Eligible riders need to provide their contactless credit or debit card information to one of our transit processors, [Littlepay](https://benefits.calitp.org/help#littlepay) or [Switchio](https://benefits.calitp.org/help#littlepay), to register their card for reduced fares.

Reviewer 2: The links are fine for now. Switchio link to be updated in #2909

Reviewer 2: ```suggestion
- Eligible riders need to provide their contactless credit or debit card information to one of our transit processors, [Littlepay or Switchio](https://benefits.calitp.org/help#payment_partners), to register their card for reduced fares.
```

Links directly to https://test-benefits.calitp.org/help#payment_partners

<img width="953" height="280" alt="image" src="https://github.com/user-attachments/assets/a3d3e938-9b17-4d9e-9685-8d23f0c5fece" />

Sensitive user information exists in the following places:

- To enroll for reduced fares as an older adult, U.S. Veteran, or Calfresh cardholder, riders need to [provide personal information to Login.gov](https://benefits.calitp.org/help#login-gov-verify).
- To enroll for reduced fares as a Medicare cardholder, riders need to [provide personal information to Medicare.gov](https://www.medicare.gov/basics/get-started-with-medicare).

> - Eligible riders need to provide their contactless credit or debit card information to one of our transit processors, [Littlepay](https://benefits.calitp.org/help#littlepay) or [Switchio](https://benefits.calitp.org/help#littlepay), to register their card for reduced fares.

Reviewer 3: ```suggestion
- Eligible riders need to provide their contactless credit or debit card information to one of our transit processors, [Littlepay](https://benefits.calitp.org/help#littlepay) or [Switchio](https://benefits.calitp.org/help#switchio), to register their card for reduced fares.
```

i think creating a generic https://benefits.calitp.org/help#transit_processors page would be a great future-proof option too.
